FourthTask/figures/main/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/tests"
+	"github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/types"
+)
+
+func main() {
+	if err := tests.Errors(); err != nil { //Programm will start work after tests check
+		fmt.Print("Everthink is ok. Program can run.\n\n")
+	}
+
+	c, err := types.CircleConstruct(5) //Create circle with side 5.
+	if err != nil {                    //Checking errors.
+		panic(err) //If we have errors, we panic. we can't go ahead.
+	}
+
+	tmp, err := c.Area()
+	if err != nil {
+		panic(err)
+	} else {
```

Comment:
`else` block can be omitted, cuz on panic code will exit. We'd want to keep code as much plain as possible.